src/main/java/Models/Prescription/Prescription.java

```diff
@@ -0,0 +1,49 @@
+package Models.Prescription;
+
+/**
+ *
+ * @author Bredan
+ */
+
+public class Prescription {
+    private final Integer patientID;
+    private final String medicine;
+    private final Integer quantity;
+    private final Boolean repeating;
+    private final String issueDate;
```

Comment:
This should be an actual Date type, not a string

Reply:
![image](https://user-images.githubusercontent.com/47354680/102346643-17646880-3f97-11eb-9e8a-a885b42d0601.png)
it is converted to date type in the PrescriptionDAO

Reply:
should be changed to a date type now in the class